root/statistics/LanguagesScripts.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2018 Shamroy Pellew
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const manifest = require('../static/manifest');
+const {l} = require('../static/scripts/common/i18n');
```

Comment:
This is actually incorrect, since these strings live in the statistics domain. We'll need to fix po/Makefile to extract things correctly too.

This might be difficult for you to fix if you don't know how our gettext setup works, though, so I can look into it later.

Reply:
https://github.com/metabrainz/musicbrainz-server/pull/622 makes the necessary changes to support this.